app/forms/claim/mentor_training_form.rb

```diff
@@ -0,0 +1,90 @@
+class Claim::MentorTrainingForm < ApplicationForm
+  attr_accessor :claim, :mentor_training, :hours_completed, :custom_hours_completed
+
+  validate :hours_completed_presence
+  validate :validate_mentor_training, unless: -> { errors.any? }
+
+  delegate :mentor_full_name, to: :mentor_training
+
+  def persist
+    updated_mentor_training.save!
+  end
+
+  def to_model
+    mentor_training
+  end
+
+  def back_path
+    if mentor_trainings.find_index(mentor_training).zero?
+      edit_claims_school_claim_mentor_path(claim.school, claim, mentor_training.mentor_id)
+    else
+      edit_claims_school_claim_mentor_training_path(
+        claim.school,
+        claim,
+        mentor_trainings[mentor_trainings.index(mentor_training) - 1],
+      )
+    end
+  end
```

Comment:
> [Array#find_index](https://docs.ruby-lang.org/en/3.2/Array.html#method-i-find_index) is an alias for [Array#index](https://docs.ruby-lang.org/en/3.2/Array.html#method-i-index).

Using both here made me wonder if they did different things.

```suggestion
  def back_path
    current_mentor_training_index = mentor_trainings.find_index(mentor_training)
    
    if current_mentor_training_index.zero?
      edit_claims_school_claim_mentor_path(claim.school, claim, mentor_training.mentor)
    else
      edit_claims_school_claim_mentor_training_path(
        claim.school,
        claim,
        mentor_trainings[current_mentor_training_index - 1],
      )
    end
  end
```